livre/core/Render/GLContext.h

```diff
@@ -43,6 +43,8 @@ class GLContext : public boost::enable_shared_from_this< GLContext >
      */
     void shareContext( GLContextPtr dstContextPtr );
 
+    const GLEWContext* glewGetContext() const;
```

Comment:
If you can show me there is a way to share a glew context with qt ( which I could not find my self on the net ), we can get rid of the whole Context API and convert to Glew based one.